### ISSUE TITLE
Config.unobserve is deprecated.

### ISSUE THREAD

**Issue by angelbotto** [AtomLinter/linter-ruby#8]
Config::unobserve no longer does anything. Call `.dispose()` on the object returned by Config::observe instead.
```
Config.unobserve (/Applications/Atom.app/Contents/Resources/app/src/config.js:1045:19)
LinterRuby.destroy (/Users/angelbotto/.atom/packages/linter-ruby/lib/linter-ruby.coffee:30:16)
```

**Comment by envygeeks** [comment]
This issue can be closed, it is resolved.

**Comment by kevinsawicki** [comment]
Fixed by #15
